src/reactNativeUtils.js

```diff
@@ -84,3 +84,20 @@ export function androidSdkVersion(): number {
   // 10 it won't be 10, it'll be 29.
   return (Platform.Version: number);
 }
+
+/**
+ * The Android user-visible version string (e.g. '13' for Android 13 Tiramisu).
```

Comment:
Based on the linked Android docs:
```suggestion
 * The Android user-visible version string, with no promised structure.
 *
 * See: https://developer.android.com/reference/android/os/Build.VERSION#RELEASE
```